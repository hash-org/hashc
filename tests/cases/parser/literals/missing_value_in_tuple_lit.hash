// run=fail, stage=parse

main := () => {
    // ~ERROR: expecting value assignment after type annotation
    t := (name : 1, data := 2)
}
