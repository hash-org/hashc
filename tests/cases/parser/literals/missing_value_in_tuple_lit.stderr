error: expected value assignment after type annotation within named tuple
 --> $DIR/missing_value_in_tuple_lit.hash:5:19
4 |       // ~ERROR: expecting value assignment after type annotation
5 |       t := (name : 1, data := 2)
  |                     ^ 
6 |   }
  = help: expected a `=`
